### PR TITLE
fix: bootstrap without waiting for promises

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
Toggles should be instantly ready (synchronous) when bootstrapping. This is needed for SSR with React SDK.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export const resolveFetch = () => {
 };
 
 export class UnleashClient extends TinyEmitter {
+    bootstrapped = false;
```
Comment (thomasheartman) [inline src/index.ts:92]: Is this essentially a combination of other fields? What does it mean if `bootstrapped` is `true`, but `bootstrap` and `bootstrapOverride` are false? Would it make more sense as a method?

And what's the property used for? I see it shows up in the test, but is it used for anything else?

File: src/index.test.ts [hunk 2]
```diff
@@ -1407,3 +1409,28 @@ test('Should be able to configure UnleashClient with a URL instance', () => {
     const client = new UnleashClient(config);
     expect(client).toHaveProperty('url', url);
 });
+
+test('Should be immediately ready when bootstrapping', () => {
```
Comment (kwasniew) [inline src/index.test.ts:1413]: this test is passing for me even before your changes so I'm wondering what is this change about?

File: src/index.ts (hunk 1 above)
Comment (kwasniew) [inline src/index.ts:92]: I don't think bootstrapped should be part of our public API of the SDK.

**Review by kwasniew — CHANGES_REQUESTED** [review]
The test that shows the new behavior is also passing for me in the old version of the SDK so something is off here.

**Pull request closed without merging**